psh/test.yaml

```
@@ -0,0 +1,11 @@
+test:
+    syspage: psh
+
+    tests:
+        - name: prompt
+          harness: test-prompt.py
+        - name: top
```

```suggestion
        - name: ps
```
I'd change test name to ps.